conf/app.routes

```diff
@@ -116,34 +116,35 @@ POST       /:taxYear/reverse-premiums-received
-GET        /:taxYear/change-private-use-adjustment                                  controllers.PrivateUseAdjustmentController.onPageLoad(taxYear: Int, mode: Mode = CheckMode)
-POST       /:taxYear/change-private-use-adjustment                                  controllers.PrivateUseAdjustmentController.onSubmit(taxYear: Int, mode: Mode = CheckMode)
-
-GET        /:taxYear/balancing-charge                                               controllers.BalancingChargeController.onPageLoad(taxYear: Int, mode: Mode = NormalMode)
-POST       /:taxYear/balancing-charge                                               controllers.BalancingChargeController.onSubmit(taxYear: Int, mode: Mode = NormalMode)
-GET        /:taxYear/change-balancing-charge                                        controllers.BalancingChargeController.onPageLoad(taxYear: Int, mode: Mode = CheckMode)
-POST       /:taxYear/change-balancing-charge                                        controllers.BalancingChargeController.onSubmit(taxYear: Int, mode: Mode = CheckMode)
-
-GET        /:taxYear/unused-residential-finance-cost                               controllers.UnusedResidentialFinanceCostController.onPageLoad(taxYear: Int, mode: Mode = NormalMode)
-POST       /:taxYear/unused-residential-finance-cost                               controllers.UnusedResidentialFinanceCostController.onSubmit(taxYear: Int, mode: Mode = NormalMode)
-GET        /:taxYear/change-unused-residential-finance-cost                        controllers.UnusedResidentialFinanceCostController.onPageLoad(taxYear: Int, mode: Mode = CheckMode)
-POST       /:taxYear/change-unused-residential-finance-cost                        controllers.UnusedResidentialFinanceCostController.onSubmit(taxYear: Int, mode: Mode = CheckMode)
-
-GET        /:taxYear/property-income-allowance                                     controllers.PropertyIncomeAllowanceController.onPageLoad(taxYear: Int, mode: Mode = NormalMode)
-POST       /:taxYear/property-income-allowance                                     controllers.PropertyIncomeAllowanceController.onSubmit(taxYear: Int, mode: Mode = NormalMode)
-GET        /:taxYear/change-property-income-allowance                              controllers.PropertyIncomeAllowanceController.onPageLoad(taxYear: Int, mode: Mode = CheckMode)
-POST       /:taxYear/change-property-income-allowance                              controllers.PropertyIncomeAllowanceController.onSubmit(taxYear: Int, mode: Mode = CheckMode)
-
-GET        /:taxYear/adjustments-start                                             controllers.AdjustmentsStartController.onPageLoad(taxYear: Int)
+# Adjustments
+GET        /:taxYear/adjustments/renovation-allowance-balancing-charge              controllers.adjustments.RenovationAllowanceBalancingChargeController.onPageLoad(taxYear: Int, mode: Mode = NormalMode)
```

Comment:
Maybe we could rearrange this routes to be in the same order as the journey? would be easier to follow